include/odp/api/spec/timer.h

```diff
@@ -214,9 +260,10 @@ odp_event_t odp_timer_free(odp_timer_t timer);
  *                            Reset operation: Too late to reset the timer.
  *
  * @see odp_timer_set_rel(), odp_timer_alloc(), odp_timer_cancel()
+ *
+ * @deprecated Use odp_timer_start() or odp_timer_restart() instead
  */
-int odp_timer_set_abs(odp_timer_t timer, uint64_t abs_tick,
-		      odp_event_t *tmo_ev);
+int odp_timer_set_abs(odp_timer_t timer, uint64_t abs_tick, odp_event_t *tmo_ev);
```

Comment:
Are odp_timer_set_* APIs needed after adding start and restart APIs?

Thanks,
Shijith

Reply:
No, but I'm planning to deprecate those in steps:
1) add new start/restart and document old set calls deprecated
2) wrap set calls inside the deprecate macro (user needs to use --enable-deprecate configure option)
3) remove calls from API (e.g. after a year from step 2)

Reply:
Okay.